modules/network-flow-matrix.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+// * installing/install_config/configuring-firewall.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="network-flow-matrix_{context}"]
+= {product-title} network flow matrix
+
+The network flow matrix describes the ingress flows to {product-title} services.
+The network information in the matrix is accurate for both bare-metal and cloud environments.
+Use the information in the network flow matrix to help you manage ingress traffic.
+You can restrict ingress traffic to essential flows to improve network security.
+
+To view or download the raw CSV content, see link:https://raw.githubusercontent.com/openshift/openshift-docs/main/snippets/network-flow-matrix.csv[this resource].
+
+Additionally, consider the following dynamic port ranges when managing ingress traffic:
+
+* `9000-9999`: Host level services
+* `3000-32767`: Kubernetes node ports
+* `49152-65535`: Dynamic or private ports
+
+[NOTE]
+====
+The network flow matrix describes ingress traffic flows for a base {product-title} installation. It does not describe network flows for additional components, such as optional Operators available from the Red Hat Marketplace. The matrix does not apply for HyperShift, MircoShift, or standalone clusters.
```

Comment:
```suggestion
The network flow matrix describes ingress traffic flows for a base {product-title} installation. It does not describe network flows for additional components, such as optional Operators available from the Red Hat Marketplace. The matrix does not apply for Hosted-Control-Plane, MicroShift, or standalone clusters.
```
"HyperShift" is not the RH-approved term, it is Hosted Control Planes. Using the hyphens here because it is a compound modifier.